### ISSUE TITLE
Results seem different

### ISSUE THREAD

**Issue by Jingnan-Jia** [jianpengz/DoDNet#19]
Hi,

I downloaded the chekcpoint and did the inference but found that the Dice of Kidney (and Kidney's tumor) is close to 0! Like the following figure shows. Did anyone meet the same issue? Can anyone help to explain the reason? Thanks a lot.

![image](https://user-images.githubusercontent.com/34941987/200292183-3780c594-3cef-49f4-86e6-a8e7bf3bf904.png)

**Comment by LiYing-hub-coder** [comment]
hello，my result:0Liver=0.7583 0LiverT=0.7583
 1Kidney=0.003324 1KidneyT=0.003324
 2Hepa=0.6531 2HepaT=0.6531
 3Panc=0.6858 3PancT=0.6858
 4ColonT=0.1915
 5LungT=0.00135
 6Spleen=0.4236
Did you solve it?

**Comment by Jingnan-Jia** [comment]
@LiYing-hub-coder I have the similar results with you. I did not solve it. I gave up. I could not reproduce the results in the paper.

**Comment by jianpengz** [comment]
The low accuracy of the Kidney may be caused by the mismatched respacing size (the old checkpoint was pre-trained under 3x0.76x0.76). So you'd better keep the same spacing size during the inference if you used the checkpoint, or you can re-train the model using the fixed spacing of 1.5x0.8x0.8.
